responsibleai/tests/causal_manager_validator.py

```diff
@@ -36,6 +37,16 @@
 
 def validate_causal(model_analysis, data, target_column,
                     treatment_features, max_cat_expansion):
+    if len(np.unique(data[target_column])) > 2:
+        with pytest.raises(AssertionError) as ae:
```

Comment:
You can use `pytest.raises(AssertionError, match="Multiclass classification isn't supported"):`

Reply:
Done.